util/xxhash3/correctness_test.go

```diff
@@ -0,0 +1,216 @@
+// Copyright 2021 ByteDance Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package xxhash3
+
+import (
+	"github.com/bytedance/gopkg/util/xxhash3/internal/xxh3_raw"
+	"math/rand"
+	"testing"
+)
+
+var dat []byte
+
+const capacity = 33554432 + 100000
```

Comment:
fell like we can use `32 * (1 << 20)` instead of `33554432`

Reply:
fixed